vmdk_plugin/main.go

```diff
@@ -97,8 +98,8 @@ func main() {
 	var driver volume.Driver
 
 	// Define command line options
-	driverName := flag.String("driver", "vmdk", "Volume driver")
-	logLevel := flag.String("log_level", "debug", "Logging Level")
+	driverName := flag.String("driver", vmdkDriver, "Volume driver")
```

Comment:
True, we can run both instances in parallel, only that adds more code that will need to be removed later. Docker on startup associates a driver to each volume that each driver reports. A volume created by a vmdk driver can get reported by a vsphere driver instance later (after docker and plugin restart) and Docker would associate that volume as a vsphere volume.

Only issue is with scripts - test scripts and user scripts to create a docker app stack that specifies the volume driver to use.
I can make driver also a part of the configuration so user can change the driver name there vs. change the startup scripts.